src/vendor/wordpress/_wordpress.scss

```diff
@@ -251,3 +251,12 @@ $wp-button-gap: size.$spacing-list-inline-gap;
   line-height: size.$height-control-multiline; /* 1 */
   padding: size.$padding-control-vertical 0;
 }
+
+/**
+ * Groups
+ * Rounds the corners for groups when the has-background class is applied.
+ */
+
+.wp-block-group.has-background {
+  border-radius: size.$border-radius-large;
```

Comment:
Do you think we should create a token specifically for card radii so the connection between the two is even more clear? (That might be overkill, just asking.)

Reply:
I wasn't going to because we already have a large border-radius with the same value and didn't want things to be too repetitive. Now that I'm looking at other tokens that doesn't seem to be a problem so I went ahead and added it.